apps/api/src/server/applications/application/application.validators.js

```diff
@@ -332,3 +332,14 @@ export const verifyNotTraining = async (caseId) => {
 		throw new Error(`Case with ID ${caseId} is a training case.`);
 	}
 };
+
+/**
+ * Is this a "Training" case? - checks the reference name and/or the sector name, rather than querying DB
+ *
+ * @param {string |undefined} reference
+ * @param {string |undefined} sector
```

Comment:
This will do the same :)
```suggestion
 * @param {string} [reference]
 * @param {string} [sector]
```

Reply:
I added the |undefined as it makes a redline ts error in the code that calls this function without it

Reply:
Oh right, even with the square brackets? No harm in keeping it in that case.

Reply:
apologies, I missed spotting the square brackets, that looks good, thanks!